src/include/ipc4/mtrace.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SOF_IPC4_MTRACE_H__
+#define __SOF_IPC4_MTRACE_H__
+
+#define SOF_MTRACE_PAGE_SIZE			0x1000
+
+int ipc4_set_enable_logs(bool first_block,
```

Comment:
Not sure if this is a good place (mtrace files) for such function because IPC4_ENABLE_LOGS in BASE_FW is used to enable all types of log modules and for probe logs we need to have IPC4_ENABLE_LOGS & IPC4_SYSTEM_TIME too.

I didnt added anything for probes here (just return 0 for IPC4_ENABLE_LOGS) at the beginning but now I think we should have a generic function to enable logs

Reply:
True, and maybe I need to rename this to something more common.

Reply:
V9 uploaded, I moved this to ipc4/logging.h. I left a TODO in comments that we need to extend IPC4_ENABLE_LOGS to support configure the type of logging.